tests/unit/data/test_plugins/openai/weathergpt.vercel.app/client/.gitignore

```diff
@@ -0,0 +1,23 @@
+__pycache__/
```

Comment:
Feels like this app shouldn't be committed, it's huge. Either it should be downloaded as part of the test or a much more minimal example should be used.